SteakBot.Core/EventHandlers/VoiceStateEventHandler.cs

```diff
@@ -1,33 +1,64 @@
-﻿using System.Linq;
-using System.Threading.Tasks;
-using Discord.WebSocket;
+﻿using Discord.WebSocket;
 using SteakBot.Core.EventHandlers.Abstraction;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 
 namespace SteakBot.Core.EventHandlers
 {
     internal class VoiceStateEventHandler : IVoiceStateEventHandler
     {
-        public async Task HandleUserVoiceStateUpdatedAsync(SocketUser user, SocketVoiceState leaveState, SocketVoiceState joinState)
+        // TODO: Get from configuration.
+        private const string FallbackMessageChannel = "bendoverwatch";
+
+        // TODO: Get from configuration.
+        private readonly IDictionary<string, string> _voiceToMessageChannelMapping = new Dictionary<string, string>
```

Comment:
Please change this to `IReadOnlyDictionary`. A good rule of thumb is to limit access to classes/methods/properties/fields as much as possible unless required otherwise.